articles/azure-functions/durable-functions-instance-management.md

```diff
@@ -111,11 +111,13 @@ public static Task Run(
 ```
 
 > [!NOTE]
-> Instance termination is currently only supported for C# functions.
+> Instance termination is currently only supported for C# orchestrator functions.
```

Comment:
Might seem that this feature is for any C# functions. Added clarification to make it know that it is for C# orchestrator functions.